Formula/kube-ps1.rb

```diff
@@ -21,7 +21,6 @@ def caveats; <<~EOS
   end
 
   test do
-    kubeon = ". #{opt_share}/kube-ps1.sh && echo $KUBE_PS1_SHELL"
-    assert_equal "zsh", shell_output("zsh -c '#{kubeon}'").chomp
```

Comment:
Are we still able to test the zsh support?

Reply:
Yes, I will add that back in, I was having issue with the characters:
```bash
/usr/local/opt/kube-ps1/share/kube-ps1.sh:27: character not in range
```

Line 27:
KUBE_PS1_SYMBOL_DEFAULT=${KUBE_PS1_SYMBOL_DEFAULT:-$'\u2388 '}

Reply:
Im stuck on capturing this output in my test.  Outside brew this test runs without fail, and actually ran without fail previously in brew.

Reply:
Hmm, how about setting `ENV["LC_CTYPE"] = "en_CA.UTF-8"` in the test before you try to invoke zsh? That fixes it in `brew test` for me.

Reply:
Perfect.  Thank you!